src/graphql/helpers.ts

```diff
@@ -114,16 +115,24 @@ export function buildWhereQuery(fields, alias, where) {
       params.push(fieldNot);
     }
 
-    const fieldIn = where[`${field}_in`] || [];
-    if (fieldIn.length > 0) {
-      query += `AND ${alias}.${field} IN (?) `;
-      params.push(fieldIn);
+    const fieldIn = where[`${field}_in`];
+    if (Array.isArray(fieldIn)) {
```

Comment:
Looks like `_in`s are always array, we don't need to check

even if we query
```
query Votes {
    spaces(first: 1000, where: { id_in: "fabien.eth" }) {
      id
    }
  }
  ```
  
  `id_in` will be an array

Reply:
I mean can be like before: 
```
    const fieldIn = where[`${field}_in`] || [];
```

Reply:
> I mean can be like before:
> 
> ```
>     const fieldIn = where[`${field}_in`] || [];
> ```

`where[`${field}_in`]` can be either 
- something valid (continue), 
- an empty array (stop), 
- or undefined (ignore)

If we cast with `|| []`, example on the space query with `id` and `created` filter:

```
spaces(first: 1000, where: { id_in: "fabien.eth" })
```

will be considered as 

```
spaces(first: 1000, where: { id_in: "fabien.eth", created_in: [] })
``` 

and will always return nothing

Reply:
Missed this point, thank you!